app/pandas.ipynb

```diff
@@ -369,9 +496,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# use * operator here"
-   ],
+   "source": "multiplication = s1 * s2",
```

Comment:
The multiplication operation is defined but not executed. You should print the result to verify the operation.